package.json

```diff
@@ -284,119 +275,114 @@
                 }
             ],
             "view/item/context": [
-                {
-                    "command": "staticWebApps.selectSubscriptions",
-                    "when": "view == staticWebApps && viewItem == azureextensionui.azureSubscription",
-                    "group": "inline"
-                },
                 {
                     "command": "staticWebApps.createStaticWebApp",
-                    "when": "view =~ /(staticWebApps|azureResourceGroups)/ && viewItem =~ /(azureextensionui.azureSubscription|azureResourceTypeGroup/microsoft.web/staticsites)/",
+                    "when": "view == azureResourceGroups && viewItem =~ /(azureextensionui.azureSubscription|azureResourceTypeGroup/microsoft.web/staticsites)/",
                     "group": "1@1"
                 },
                 {
                     "command": "staticWebApps.createStaticWebAppAdvanced",
-                    "when": "view =~ /(staticWebApps|azureResourceGroups)/ && viewItem == azureextensionui.azureSubscription",
+                    "when": "view == azureResourceGroups && viewItem == azureextensionui.azureSubscription",
```

Comment:
Some of these have exact equality on the `viewItem`; are all of them sub-nodes under the main SWA node (so, owned by the SWA extension), or might some need to change to regexs?